framer.py

```diff
@@ -18,19 +18,24 @@ def __init__(self):
 		self.fontSize = self.config["fontsize"]
 		self.xPos = str(self.config["xposition"])
 		self.yPos = str(self.config["yposition"])
+		self.folder = self.config["output"]
```

Comment:
self.folder is the folder of source images. So it should not be changed. There must be another self.output parameter.